ncscolor.js

```diff
@@ -116,7 +116,9 @@
         top = Gc
       } else if (Bc > Rc && Bc > Gc) {
         top = Bc
-      } else {
+      } else if (Rc == Gc) {
```

Comment:
So this library has a pretty twisted history: some time around 2008 I needed this functionality in a Flash project and ported some random Java Code I found on a forum. At some later point I ported the Flash project to JavaScript and this repo got created. This means the code is as twisted as it is right now.

Looking at this with a little more distance I wonder if this pretty complicated procedure couldn't simply be:

```
top = Math.max(Rc, Bc, Gc)
```

because we basically just want the biggest of these values?

Reply:
tested with my `compare-ncscolor-vs-w3color.js` and the result is the same : )